script/autopush.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
Nit: can we rename this folder to 'scripts'? It will probably grow to hold more scripts in the future

Reply:
Done